CHANGELOG.md

```diff
@@ -19,3 +19,11 @@
 - Add: show number of validators in tabs title (validators page) ([\#68](https://github.com/forbole/big-dipper-2.0-solana/issues/68))
 - Add tokens chart ([\#76](https://github.com/forbole/big-dipper-2.0-solana/issues/76))
 - Add Stake Weight Component ([\#41](https://github.com/forbole/big-dipper-2.0-solana/issues/41))
+- Fix: update general chain config report issue url ([\#98](https://github.com/forbole/big-dipper-2.0-solana/issues/98))
+- Update: hash and messages to signature and instruction in transaction sections ([\#103](https://github.com/forbole/big-dipper-2.0-solana/issues/103))
+- Update: dockerfile and .env names ([\#89](https://github.com/forbole/big-dipper-2.0-solana/issues/89))
+- Home: Fix overflowing in latest blocks responsive views ([\#85](https://github.com/forbole/big-dipper-2.0-solana/issues/85))
+- Remove tx style option in global settings ([\#83](https://github.com/forbole/big-dipper-2.0-solana/issues/83))
+- Update 404 and not found logo ([\#99](https://github.com/forbole/big-dipper-2.0-solana/issues/99))
+- Update gql ([\#100](https://github.com/forbole/big-dipper-2.0-solana/issues/100))
+- Fix AvatarName if name === '""' or length is empty return address ([\#95](https://github.com/forbole/big-dipper-2.0-solana/issues/95))
```

Comment:
For future reference please just focus on your own issue :0. I sometimes don't add it because I don't think it's necessary

Reply:
I see. I thought you forgot so added everything 😅 Should I remove them and only leave this issue number?

Reply:
I will remove~